spikeextractors/extractors/openephysextractors/openephysextractors.py

```diff
@@ -19,15 +19,20 @@ class OpenEphysRecordingExtractor(RecordingExtractor):
     mode = 'folder'
     installation_mesg = "To use the OpenEphys extractor, install pyopenephys: \n\n pip install pyopenephys\n\n"  # error message when not installed
 
-    def __init__(self, folder_path, *, experiment_id=0, recording_id=0, dtype='float'):
+    def __init__(self, folder_path, *, experiment_id=0, recording_id=0, dtype='int16'):
```

Comment:
Hi @luiztauffer 

I'd keep `float` as default, but add a `dtype` arg to the `get_traces` to allow for dynamically select the dtype.

Note that these changes will be soon superseeded by the `return_scaled` arg.